pythonforandroid/recipes/numpy/__init__.py

```diff
@@ -1,6 +1,7 @@
-from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe, sh, info, current_directory, shprint
```

Comment:
Prefer importing from the true origin module, `sh` is its own thing (`import sh`), the others come from pythonforandroid.logger etc.